spec/defines/selinux_exec_restorecon_spec.rb

```diff
@@ -2,10 +2,14 @@
 
 describe 'selinux::exec_restorecon' do
   on_supported_os.each do |os, _facts|
+
+
     context "on #{os}" do
       context 'with resource titled /opt/mycustompath' do
         let(:title) { '/opt/mycustompath' }
-
+        let :facts do
```

Comment:
You can do this at a higher level (just below `context "on #{os}" do`) and just do it once